src/components/v5/shared/CountBox/CountBox.tsx

```diff
@@ -0,0 +1,11 @@
+import React, { type FC } from 'react';
+
+import { type CountBoxProps } from './types.ts';
+
+const CountBox: FC<CountBoxProps> = ({ count }) => (
+  <span className="flex items-center justify-center text-blue-400 text-[0.5rem] font-bold bg-blue-100 rounded-sm ml-1.5 w-[0.75rem] h-[0.75rem] pointer-events-none">
```

Comment:
`w-[0.75rem]` and `h-[0.75rem]` can be replaced by `w-3` and `h-3`

Reply:
I had to change it to min-width